scenarios/kubernetes_bazel.py

```diff
@@ -50,17 +54,80 @@ def get_version():
     with open('bazel-genfiles/version') as fp:
         return fp.read().strip()
 
+def get_affected(base, pull):
```

Comment:
nit: "changed" is probably a more correct name than "affected" - affected are the packages returned by the rdeps bazel query.